### PR TITLE
♻️ Removes `opt_` prefixes from param names - also updates code style a bit

### PR DESCRIPTION
**Pull request by ChrisAntaki**
- Removes `opt_` prefixes from param names
- Updates a few `var/let` declarations to `let/const`
- Automatic formatting updates from `prettier`

### CODE REVIEW

File: src/api/entitlements.js [hunk 1]
```diff
@@ -54,13 +58,14 @@ export class Entitlements {
...
   clone() {
     return new Entitlements(
-        this.service,
-        this.raw,
-        this.entitlements.map(ent => ent.clone()),
-        this.product_,
-        this.ackHandler_,
-        this.isReadyToPay,
-        this.decryptedDocumentKey);
+      this.service,
```
Comment (dparikh) [inline src/api/entitlements.js]: should this be 4 chars indentation, since it is a continuation line?

File: src/api/entitlements.js [hunk 2]
```diff
@@ -217,8 +227,10 @@ export class Entitlement {
     }
     // Wildcard allows this product.
     const eq = product.indexOf(':');
-    if (eq != -1 &&
-        this.products.includes(product.substring(0, eq + 1) + '*')) {
+    if (
```
Comment (dparikh) [inline src/api/entitlements.js]: I would prefer the previous formatting.

File: build-system/tasks/jsify-css.js [hunk 3]
```diff
@@ -56,17 +56,16 @@ const cssnano = cssnanoDecl({
...
  *
  * @param {string} filename css file
- * @param {?Object=} opt_options
+ * @param {?Object=} options
  * @return {!Promise<string>} that resolves with the css content after
  *    processing
  */
-exports.jsifyCssAsync = async (filename, opt_options) => {
-  const options = Object.assign(
+exports.jsifyCssAsync = async (filename, options) => {
```
[the remark this answers is not in the document]
Comment (ChrisAntaki) [reply build-system/tasks/jsify-css.js]: Sure, that seems clearer

File: src/api/entitlements.js [hunk 4]
```diff
@@ -29,9 +27,15 @@ export class Entitlements {
    * @param {?boolean|undefined} isReadyToPay
    * @param {?string|undefined} decryptedDocumentKey
    */
-  constructor(service, raw, entitlements, currentProduct, ackHandler,
-    isReadyToPay, decryptedDocumentKey) {
-
+  constructor(
```
[the remark this answers is not in the document]
Comment (ChrisAntaki) [reply src/api/entitlements.js]: This formatting was automatically done by `prettier`

File: src/api/entitlements.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/api/entitlements.js]: This formatting was automatically done by `prettier`

File: src/api/entitlements.js (hunk 2 above)
Comment (ChrisAntaki) [reply src/api/entitlements.js]: This formatting was automatically done by `prettier`

**Comment by ChrisAntaki** [comment]
Thanks for the review!

**Comment by jpettitt** [comment]
please make sure this still builds with amp in light of https://github.com/ampproject/amphtml/search?q=opt_&unscoped_q=opt_

**Comment by ChrisAntaki** [comment]
Yep! This PR just made our rules a bit stricter, they'll still work with AMP's though

**Pull request merged**